Tests/UnitTests/Shared/Shared.cpp

```diff
@@ -23,36 +24,92 @@ const char* EnumToString(Babylon::Polyfills::Console::LogLevel logLevel)
     return "unknown";
 }
 
-int RunTests(Babylon::Polyfills::Console::CallbackT consoleCallback)
+TEST(JS, JSTests)
 {
     std::promise<int32_t> exitCode;
 
     Babylon::AppRuntime runtime{};
 
-    runtime.Dispatch([&exitCode, consoleCallback = std::move(consoleCallback)](Napi::Env env) mutable
-    {
+    runtime.Dispatch([&exitCode, consoleCallback = std::move([](const char* message, Babylon::Polyfills::Console::LogLevel logLevel) {
+        fprintf(stdout, "[%s] %s", EnumToString(logLevel), message);
+        fflush(stdout);
+    })](Napi::Env env) mutable {
         Babylon::Polyfills::XMLHttpRequest::Initialize(env);
         Babylon::Polyfills::Console::Initialize(env, std::move(consoleCallback));
         Babylon::Polyfills::Scheduling::Initialize(env);
         Babylon::Polyfills::URL::Initialize(env);
         Babylon::Polyfills::AbortController::Initialize(env);
         Babylon::Polyfills::WebSocket::Initialize(env);
 
-        env.Global().Set("SetExitCode", Napi::Function::New(env, [&exitCode](const Napi::CallbackInfo& info)
-        {
-            Napi::Env env = info.Env();
-            exitCode.set_value(info[0].As<Napi::Number>().Int32Value());
-        }, "SetExitCode"));
+        env.Global().Set("SetExitCode", Napi::Function::New(
+                                            env, [&exitCode](const Napi::CallbackInfo& info) {
+                                                Napi::Env env = info.Env();
+                                                exitCode.set_value(info[0].As<Napi::Number>().Int32Value());
+                                            },
+                                            "SetExitCode"));
```

Comment:
nit: Is this formatting coming from clang-format? I thought our formatting doesn't do this indenting.

Reply:
fixed